src/com/lftechnology/java/training/niraj/stemleaf/StemLeafPlot.java

```diff
@@ -0,0 +1,88 @@
+package com.lftechnology.java.training.niraj.stemleaf;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.alina.basic.SumOfMultiples;
+
+/**
+ * 
+ * Plots stem and leaf
+ * 
+ * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+ *
+ */
+public class StemLeafPlot {
+	private final static Logger LOGGER = Logger.getLogger(SumOfMultiples.class
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)